src/components/SubNavigation.js

```diff
@@ -2,46 +2,49 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { css } from '@emotion/core';
 import { Link } from 'gatsby';
-import { NavItem, Icon } from '@newrelic/gatsby-theme-newrelic';
+import { NavItem, Icon, useTranslation } from '@newrelic/gatsby-theme-newrelic';
 
-const SubNavigation = ({ nav }) => (
-  <>
-    <Link
-      to="/"
-      css={css`
-        color: var(--primary-text-color);
-        display: flex;
-        align-items: center;
-        padding: 0.5rem 0;
-        margin-bottom: 0.5rem;
-        transition: 0.2s ease-out;
-
-        &:hover {
-          color: var(--primary-text-hover-color);
-        }
-      `}
-    >
-      <Icon
-        name="fe-home"
-        size="1rem"
+const SubNavigation = ({ nav }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <Link
+        to="/"
         css={css`
-          margin-right: 0.5rem;
+          color: var(--primary-text-color);
+          display: flex;
+          align-items: center;
+          padding: 0.5rem 0;
+          margin-bottom: 0.5rem;
+          transition: 0.2s ease-out;
+
+          &:hover {
+            color: var(--primary-text-hover-color);
+          }
         `}
-      />
-      Home
-    </Link>
-    {nav && (
-      <>
-        <h2>{nav.title}</h2>
-        <nav role="navigation" aria-label="SubNavigation">
-          {nav.pages.map((page) => (
-            <NavItem key={page.title} page={page} />
-          ))}
-        </nav>
-      </>
-    )}
-  </>
-);
+      >
+        <Icon
+          name="fe-home"
+          size="1rem"
+          css={css`
+            margin-right: 0.5rem;
+          `}
+        />
+        {t('subNav.title')}
```

Comment:
```suggestion
        {t('subNav.homeLink')}
```

This home link isn't really a title of the sidenav, but rather is just a way for the user to get back to the home page. You can see the use of `nav.title` on [line 37](https://github.com/newrelic/docs-website/pull/536/files#diff-e93720bfaaf6d5f84c5bec74dd0fdfee1b17ed461f41596406f7c83093ddd718R37) which makes this name a bit confusing.

Reply:
ah yep. not sure why i reached for the word `title` TBH. good catch